### PR TITLE
Add routes for issues and pr views.

### PR DESCRIPTION
**Pull request by mariosant**

### CODE REVIEW

File: client/components/issuesIndex/issuesIndex.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Template.issuesIndex.helpers({
+  example: function() {
```
Comment (mariosant) [inline client/components/issuesIndex/issuesIndex.js:2]: This is an example of how to use router's parameters on helpers.

**Comment by howardroark** [comment]
Can we have it all use the same view?  Sort of assume that everything has the same schema and that it is more about filtering.  The router can do the job of getting the schema ready and figuring out what is what.

user/repo = all
user/repo/issues = issues
user/repo/pulls = pulls

We could have another view that is for a 'detailed' summary.

Is that a meteor friendly option?  Just keeps it more simple for dealing with UI :)

**Comment by mariosant** [comment]
Sure it is! Will modify the PR now.

**Comment by mariosant** [comment]
https://github.com/mariosant/gitsup/commit/9ac3714fad04c8e2f9160a1c7e27ffa7e8cea0d3 should do it @howardroark !

**Comment by howardroark** [comment]
Looks good!  I do like the term `project` vs `repo`, makes more sense.

**Comment by mariosant** [comment]
I didn't really think about it! Are you sure it does?

On Sat, 12 Mar 2016 21:11 Howie, notifications@github.com wrote:

> Looks good! I do like the term project vs repo, makes more sense.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gitsup/gitsup/pull/11#issuecomment-195792857.
> 
> ## 

Marios Antonoudiou

**Comment by howardroark** [comment]
Hah! Yeah the project is sort of the whole thing.  The repo is about the commits.

**Pull request merged**